tests/sentry/tasks/test_commit_context.py

```diff
@@ -581,6 +632,7 @@ def test_gh_comment_repeat_issue(self, get_jwt, mock_comment_workflow):
                 project_id=self.event.project_id,
             )
             assert not mock_comment_workflow.called
+            assert len(PullRequestCommit.objects.all()) == 0
```

Comment:
nit: I'd recommend we use `PullRequestCommit.objects.all().count() == 0` here instead -> Django will optimize the SQL query to just get the number of results, not the actual results themselves ([source](https://stackoverflow.com/questions/10695322/django-show-the-length-of-a-queryset-in-a-template)). 

It doesn't really matter here since the count will be so small, but it won't hurt.